### PR TITLE
[dom] Add GREASY 18.07

### PR DESCRIPTION
**Pull request by victorusu**
There are two versions of GREASY, one with GPU support and the other without

### CODE REVIEW

**Comment by victorusu** [comment]
retest this please

**Comment by jgphpc** [comment]
retest this please

**Comment by jgphpc** [comment]
18.07 => change the title to [dom-gpu]

**Comment by victorusu** [comment]
retest this please

File: easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-18.07-gpu.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+homepage = 'https://github.com/jonarbo/GREASY'
+description = """Greasy is a tool designed to make easier the deployment of
+embarrassingly parallel simulations in any environment. It is able to run in
+parallel a list of different tasks."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.07'}
+toolchainopts = {'opt': True, 'pic': True}
+
+sources = ['/apps/common/easybuild/sources/g/GREASY/greasy-%(version)s-gpu.tar.gz']
```
Comment (gppezzi) [inline easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-18.07-gpu.eb]: If you don't use sources from a git repo, could you please add as comment where does this version come from? (same for non-gpu version). If you modified the sources, maybe you should fetch it from your repository?

Comment (jgphpc) [reply easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-18.07-gpu.eb]: https://github.com/eth-cscs/production/pull/692/commits/f768b898eaea3e15e04fcd5a762b679e65e5d82d

Comment (jgphpc) [reply easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-18.07-gpu.eb]: done

Comment (jgphpc) [reply easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-18.07-gpu.eb]: fixes https://github.com/eth-cscs/production/pull/692#pullrequestreview-143877649

**Comment by jgphpc** [comment]
will use madra as repo for the src + update to 18.08

**Pull request merged**